pipeline/test_beam_tables.py

```diff
@@ -24,26 +24,36 @@
 
 from pipeline import beam_tables
 from pipeline.metadata.ip_metadata_chooser import FakeIpMetadataChooserFactory
+from pipeline.metadata import flatten_hyperquack
 from pipeline.metadata import satellite
 
 
 class PipelineMainTest(unittest.TestCase):
   """Unit tests for beam pipeline steps."""
 
+  def setUp(self) -> None:
+    self.maxDiff = None  # pylint: disable=invalid-name
+
   # pylint: disable=protected-access
 
-  def test_get_bigquery_schema(self) -> None:
+  def test_get_bigquery_schema_hyperquack(self) -> None:
     """Test getting the right bigquery schema for data types."""
     echo_schema = beam_tables._get_bigquery_schema('echo')
-    self.assertEqual(echo_schema, beam_tables.SCAN_BIGQUERY_SCHEMA)
+    all_hyperquack_top_level_columns = (
+        list(beam_tables.BASE_BIGQUERY_SCHEMA.keys()) +
+        list(flatten_hyperquack.ADDITIONAL_HYPERQUACK_BIGQUERY_SCHEMA.keys()))
+    self.assertListEqual(
+        list(echo_schema.keys()), all_hyperquack_top_level_columns)
 
+  def test_get_bigquery_schema_satellite(self) -> None:
     satellite_schema = beam_tables._get_bigquery_schema('satellite')
     all_satellite_top_level_columns = (
-        list(beam_tables.SCAN_BIGQUERY_SCHEMA.keys()) +
-        list(satellite.SATELLITE_BIGQUERY_SCHEMA.keys()))
+        list(beam_tables.BASE_BIGQUERY_SCHEMA.keys()) +
```

Comment:
I'd rather see here something like `HYPERQUACK_BIGQUERY_SCHEMA.keys()`.

It's a lot more readable. You are leaking how the schema is built, but the reader shouldn't have to care.

Reply:
done